pegasus/rake/generate_pdfs.rake

```diff
@@ -3,6 +3,7 @@ require 'cdo/hip_chat'
 require 'cdo/rake_utils'
 require 'cdo/tempfile'
 require 'pdf/conversion'
+require 'cdo/properties'
```

Comment:
@laurelfan hopefully it's okay to be reaching back to cdo and the pegasus database from this rake file.